playbook-website/app/controllers/guides_controller.rb

```diff
@@ -30,8 +32,17 @@ def set_page_vars
            else
              Dir.glob("#{Dir[search_path].first}*.md").first
            end
-    @link_extension = File.basename(file)
-    @front_matter = render_frontmatter(file)
-    @page_title = @front_matter["title"] || File.basename(file, ".*").split(".")[0].tr("_", " ")
+    if file == "README.md"
+      file = nil
+      @parent_not_found = true
+    end
+    if file
+      @link_extension = File.basename(file)
+      @front_matter = render_frontmatter(file)
+      @page_title = @front_matter["title"] || File.basename(file, ".*").split(".")[0].tr("_", " ")
+      @guide_exists = true
+    else
+      @guide_exists = false
+    end
```

Comment:
Would it make more sense to combine these into a single if/else statement?

```suggestion
    if file == "README.md"
      @parent_not_found = true
      @guide_exists = false
    elsif file
      @link_extension = File.basename(file)
      @front_matter = render_frontmatter(file)
      @page_title = @front_matter["title"] || File.basename(file, ".*").split(".")[0].tr("_", " ")
      @guide_exists = true
    else
      @guide_exists = false
    end
```

Reply:
Nice, much cleaner. I'll go ahead and add this- thank you!